.project

```diff
@@ -15,15 +15,9 @@
 			<arguments>
 			</arguments>
 		</buildCommand>
-		<buildCommand>
-			<name>net.sf.eclipsecs.core.CheckstyleBuilder</name>
-			<arguments>
-			</arguments>
-		</buildCommand>
 	</buildSpec>
 	<natures>
 		<nature>org.eclipse.buildship.core.gradleprojectnature</nature>
 		<nature>org.eclipse.jdt.core.javanature</nature>
-		<nature>net.sf.eclipsecs.core.CheckstyleNature</nature>
 	</natures>
```

Comment:
What I did so far was to deactivate it and configure git to ignore the `.project` file at the personal level by [doing this](
https://stackoverflow.com/questions/6117109/few-questions-with-git-possible-to-have-custom-gitignore-read-only-access). Most of the time `.project` is ignored on my side but sometimes CheckStyle gets magically activated again.